app/controllers/trainees/language_specialisms_controller.rb

```diff
@@ -12,10 +12,8 @@ def edit
     def update
       @language_specialisms_form = LanguageSpecialismsForm.new(trainee, params: language_specialism_params, user: current_user)
 
-      save_strategy = trainee.draft? ? :save! : :stash
```

Comment:
Are we happy with not saving directly to a draft trainee? Just thinking this is different to all the other forms we have.. does it have and Bad Consequences?

Reply:
We could... The problem this was causing is that `save!` was wiping the stash. That would lead to a strange scenario where we would be using a combination of stashed/saved attributes in the `ConfirmPublishCourse` form